### PR TITLE
solution basic calc tests

### PR DESCRIPTION
**Pull request by tarasulo**

### CODE REVIEW

File: src/main/java/core/basesyntax/Calc.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Calc {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Calc.java]: Use full name of the class

File: src/test/java/core/basesyntax/CalcTest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CalcTest {
```
Comment (Sofasmile) [inline src/test/java/core/basesyntax/CalcTest.java]: the same about name

File: src/test/java/core/basesyntax/CalcTest.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+    private Calc calc = null;
```
Comment (Sofasmile) [inline src/test/java/core/basesyntax/CalcTest.java]:
```suggestion
    private Calc calc;
```

File: src/test/java/core/basesyntax/CalcTest.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+    @BeforeClass
+    public static void onceExecutedBeforeAll() {
```
Comment (Sofasmile) [inline src/test/java/core/basesyntax/CalcTest.java]: do we need this method here?

File: src/test/java/core/basesyntax/CalcTest.java [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        System.out.println("@BeforeClass: onceExecutedBeforeAll");
+    }
+
+    @Before
+    public void executedBeforeEach() {
+        calc = new Calc();
+        System.out.println("@Before: executedBeforeEach");
```
Comment (Sofasmile) [inline src/test/java/core/basesyntax/CalcTest.java]: remove sout

File: src/main/java/core/basesyntax/Calc.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    public static double total(double a, double b, char operation) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Calc.java]:
```suggestion
    public static double calculate(double a, double b, char operation) {
```

File: src/test/java/core/basesyntax/CalcTest.java [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+        Assert.assertEquals(0, actual, 0);
+        actual = calc.total(3, 1.5, '/');
+        Assert.assertEquals(2, actual, 0);
+    }
+
+    @Test
+    public void validPow() {
+        double actual = calc.total(2, 2, '^');
+        Assert.assertEquals(4, actual, 0);
+        actual = calc.total(4, -1, '^');
```
Comment (Sofasmile) [inline src/test/java/core/basesyntax/CalcTest.java]: What about (5, 0, '^')?

Comment (tarasulo) [reply src/test/java/core/basesyntax/CalcTest.java]: done

**Pull request closed without merging**